pkg/querier/tail/metrics.go

```diff
@@ -1,18 +1,18 @@
-package querier
+package tail
 
 import (
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promauto"
 )
 
-type Metrics struct {
+type TailMetrics struct {
```

Comment:
These should be called `Metrics` like before, see linter error.

Reply:
yep will do